### ISSUE TITLE
ImageCell with text

### ISSUE THREAD

**Issue by MatKier** [dhorions/boxable#50]
At the moment it isn't possible to write text into an ImageCell, or at least I couldn't do it. 
It would be nice to place a text above, under or directly onto a picture.

Do you plan on adding this feature in the future? / Would you consider adding this?

**Comment by Frulenzo** [comment]
This would be nice feature to have in feature but images with text in the cell with proper alignment is royal pain :smile:

**Comment by andreabtr95** [comment]
Hi, i didn't know that exists this library, and like you i made my library based on PDFBox in these 6 months. However i have some problems with the splitting of tables. But in my library i found a way to make "ImageLabel", image with text, and text with multi-style and color inside a cell. If you want to help me, and to help yourself, contact me.

**Comment by dhorions** [comment]
Hi @andreabtr95 , this is certainly interesting.
Can you share a link to your code so I can have a look at it.

**Comment by Frulenzo** [comment]
I am also very interested in your solution with this problem. If you hesitate to post in this thread can you visit our [gitter channel](https://gitter.im/dhorions/boxable?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) and tell us more about your project?

**Comment by Frulenzo** [comment]
Hi @andreabtr95 again. Can you please share your code/ideas about this problem. That would be really helpful and deeply appreciated :wink:

**Comment by Frulenzo** [comment]
Hi @andreabtr95 . I tried to visit your linkedin page for your email so I don't spam this thread but unfortunately I get 404. Can you please (pretty please :smile: ) share your code/ideas with this particular feature request.

**Comment by andreabtr95** [comment]
Hi @Frulenzo . I tried to check if i can share my code because i did that for my company. Hovewer if you want i can explain you the concept of my solution.

**Comment by Frulenzo** [comment]
Yeah, please do. If you are unwilling to do that here please email me at: stimac@szue.de

**Comment by Frulenzo** [comment]
@andreabtr95 any new info about this?

**Comment by Frulenzo** [comment]
@andreabtr95 small *bump* :smile:

**Comment by quodlibetbv** [comment]
closing due to no activity.
